src/Suave/Owin.fs

```diff
@@ -333,13 +345,14 @@ module OwinAppFunc =
 
     let steamyBytes : Property<HttpContent, IO.Stream> =
       (fun x ->
-        let ms = new IO.MemoryStream()
+        // TODO: provide a Stream that wraps the following:
+        //let impl conn = AsyncSocket.transferStream conn v
+        //SocketTask impl
+        // TODO: may also provide a Stream that continuously writes to the HttpContext.
         let bs = Lens.getPartialOrElse HttpContent.BytesPLens [||] x
-        ms.Write(bs, 0, bs.Length) // hmm? flag that we have a stream?
-        upcast ms),
-      (fun v x ->
-        let impl conn = AsyncSocket.transferStream conn v
-        SocketTask impl)
```

Comment:
I assume that the above two lines trigger sending headers, as well as fires the `OnSendingHeaders` callback.